Dockerfile.assisted-iso-create

```diff
@@ -0,0 +1,22 @@
+#This image contains the RHCOS livecd.iso
+FROM quay.io/ocpmetal/livecd-iso:rhcos-livecd AS livecd-build
+RUN chmod 666 /root/image/livecd.iso
+
+FROM quay.io/coreos/coreos-installer:release AS installer-image
+
+FROM python:3-slim
+
+ARG WORK_DIR=/data
+ENV COREOS_IMAGE=$WORK_DIR/livecd.iso
+ENV WORK_DIR=$WORK_DIR
+
+RUN mkdir $WORK_DIR
+RUN chmod 777 $WORK_DIR
+
+COPY --from=livecd-build /root/image/livecd.iso $WORK_DIR
+COPY --from=installer-image /usr/sbin/coreos-installer $WORK_DIR
+
+COPY build/assisted-iso-create $WORK_DIR
+
+ENV COMMMAND_LINE=$WORK_DIR/assisted-iso-create
+ENTRYPOINT exec $COMMMAND_LINE
```

Comment:
Can we run as non-root? See this PR for an example: https://github.com/openshift/assisted-service/pull/74

Reply:
yes, the changes to make it run as a non-root we already implemented in the coreos_installer_iso repo, i just moved them here